types/react-native-auth0/index.d.ts

```diff
@@ -316,7 +316,7 @@ export class Auth0ContextInterface {
     user: any;
     error: any;
     authorize(parameters: AuthorizeParams, options?: AuthorizeOptions): Promise<void>;
-    clearSession(): Promise<void>;
+    clearSession(parameters?: ClearSessionParams): Promise<void>;
```

Comment:
It's good that you complete the signature, but you might want to make it fully complete by also adding the second method parameter (options).

Reply:
And if you do, also add them to `export class WebAuth`